app/helpers/application_helper.rb

```diff
@@ -16,6 +16,13 @@ def tooltip(text)
     )
   end
 
+  def alert_category(key)
+    key = key.to_s.downcase
+    return 'oops' if key.in? %w(error alert)
+    return 'success' if key.in? %w(notice success)
```

Comment:
i think that's fine -- i was worried that devise was auto-creating some flash messages with diff flash types that that may not be true; will explore!